assemblies/pad-ce/src/main/resources/plugins/readme.txt

```diff
@@ -2,4 +2,4 @@ Place any extensions to the AggDesigner in this directory.
 Please see the Plugin Developers Guide for more information 
 on extending AggDesigner.
 
-http://wiki.pentaho.com/AggDesigner/Plugin-Developers-Guide
+https://pentaho-community.atlassian.net/wiki/AggDesigner/Plugin-Developers-Guide
```

Comment:
I'm getting a page not found for this link.

Reply:
I couldn't find what looked like a correct link, even browsing wayback machine. Probably better to just link to the base page of the wiki.